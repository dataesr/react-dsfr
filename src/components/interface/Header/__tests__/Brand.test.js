import renderer from 'react-test-renderer';
import {
  Brand,
} from '../index';

describe('<Brand />', () => {
  it('renders correctly', () => {
    const component = renderer
      .create(
        <Brand
          title="title"
          linkTitle="République française"
        />,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
