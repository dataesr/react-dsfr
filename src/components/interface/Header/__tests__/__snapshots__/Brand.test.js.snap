// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Brand /> renders correctly 1`] = `
<div
  className="rf-header__brand"
>
  <a
    className="rf-logo"
    href="/"
    title="République française"
  >
    <span
      className="rf-logo__title"
      dangerouslySetInnerHTML={
        Object {
          "__html": "title",
        }
      }
    />
  </a>
</div>
`;
